devon4j-app/index.asciidoc

```diff
@@ -0,0 +1,38 @@
+= devon4j application generation
+====
+Devon4j is the Java stack of devonfw. It allows you to build business applications (backends) using Java technology in a highly efficient and standardized way based on established best-practices.
+
+More information about Devon4j on https://devonfw.com/website/pages/docs/devon4j.asciidoc.html
+
+= Project structure
+====
```

Comment:
You have to use some other delimiter here than '===='. This is the delimiter for the description part. So the text after this section will not be displayed in the intro part in katacoda.